unicorn_properties/tests/events/eventbridge/contract_status_changed_event_contract_1_draft.json

```diff
@@ -2,7 +2,7 @@
   {
     "DetailType": "ContractStatusChanged",
     "Source": "unicorn.contracts",
-    "EventBusName": "UnicornPropertiesEventBus-Local",
```

Comment:
This is the right event bus name the event flow follow: Unicorn Contracts's DynamoDB -> DynamoDB CDC stream -> Unicorn Properties event bus -> Unicorn Properties event handler.
So the the event will be coming from Unicorn Properties's event bus.